### PR TITLE
Modified CSV's test and samples

### LINKED ISSUE #481: Updation of Test CSV Files
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Expected behavior:**
CSV files and it usage should be updated according to #476 

I am working on this Issue.

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #481 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Updated CSV Samples according to changes done in #476 
- Modified CSV tests

### CODE REVIEW

**Comment by ParthS007** [comment]
@harshit98 @vaibhavsingh97 @abishekvashok Please review .Thanks

File: README.md [hunk 1]
```diff
@@ -55,21 +55,12 @@ The guidelines for setting up and running the tests are mentioned in the [testin
 
 - The input is a set of csv files in the same folder, UTF-8.
 - The csv file is named after the badge type to take. Example: `vip.png.csv` uses the picture `vip.png`.
-- The CSV has up to 4 columns for the name and the twitter handle. They will be filled if this number is filled:
-  - `__X_`
-  - `__XX`
-  - `_XXX`
-  - `XXXX`
-- Optional configuration file in json format to customise badges.
-- A sample configuration file is shown below
```
Comment (rupav) [inline README.md]: This got deleted too, needs to be restored.

Comment (mohdomama) [reply README.md]: Looks good @ParthS007 :+1: 
Resolve this and then LGTM!

Comment (rupav) [reply README.md]: Ok now that JSON file upload option is removed, its not required. LGTM then.

**Comment by abishekvashok** [comment]
Thank you @ParthS007 :)
@rupav @mohdomama thanks for the reviews

**Pull request merged**